### ISSUE TITLE
Bug report for GX and SFT versions of Wii64 (R474)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#77]
```
What steps will reproduce the problem?
1. Compile "GX and SFT" versions of wii64
2. run via HBC
3. Tried several games (listed below)

What is the expected output? What do you see instead?
Just listing problems I found and what I observed.

What version of the product are you using? On which system and loader?
R474 - GX and R474 - SFT

Please provide any additional information below.
Diddy Kong racing - Crashes before starting with what appears to be an 
unknown instruction (unknown opcode) error. appears to perform memory dump.
Both versions.

Duke nukem 64 - GX - no control over the classic controller or GC 
controller but runs at a pretty decent speed.
SFT - Crashes and does memory dump.

Goldeneye - GX - Runs very slow
SFT - Black screen

Mortal kombat trilogy - GX - Black screen
SFT - Black screen

If you want or need more info feel free to e-mail me.

```

Original issue reported on code.google.com by `allterra...@aol.com` on 5 Jun 2009 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
The GX and Soft gfx plugins will not be worked on any further, glN64_GX is the 
new
standard.
```

Original comment by `emuki...@gmail.com` on 30 Sep 2009 at 1:31
- Changed state: **WontFix**
